_posts/2019-11-01-printer-setup.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: How To Setup Your Printer
+date: 2019-11-01 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+# Printer Setup Still Sucks
+
+>"Just click a button on the printer and it will do everything itself"
+
+Nothing could be further from the truth.
+
+It is 2019 and printers still suck. You would have thought that by now someone, anyone, would have come out with a decent printer. Nope, they all suck. After the initial setup it is bearable and there are a lot of good printers that work. Yes, the expectations are so low that a printer that simply prints without issues on a regular basis has to be called good.
+
+I had to first connect the printer via usb since the automatic on board setup did not work. After connecting it I had to manually go the manifacturers website to download the drivers or you could also use a qr code. This is a printer for businesses so no idea who would use a qr code from a desktop machine I wonder...?
```

Comment:
manifacturers - > manufacturers

Reply:
Plus add no way we use the cd since no laptop has a drive (there was a CD with drivers)